### DISCUSSION TITLE
h2o_wave vs. panel vs. streamlit, etc.

### DISCUSSION THREAD

**Discussion by bryanwhiting** [h2oai/wave#1789] [General]
Hello!

I'm sure this will start a 🔥 ... I was reading this: https://medium.com/@marcskovmadsen/i-prefer-to-use-panel-for-my-data-apps-here-is-why-1ff5d2b98e8f and really benefitted from comparing panel to streamlit.

One thing I love about panel's design is how easy it is to get something up and running (without using all the pages/cards). I love h2o_wave because it looks beautiful. I get the feeling that wave would be faster than its competitors.

Curious to hear others' feedback on why they chose wave vs. other tools?

**Comment by mturoci** [comment]
Leaving this up to the community to answer (although it might not get a lot of traction here).

The [general answer](https://discord.com/channels/1022255416454946906/1022255416454946909/1058159619333160960) as a core team is :

While we don't have an official comparison, the general thing we say to our sales people is: Look at app showcase (at wave.h2o.ai) and ask yourself if these kinds of apps would be possible to do in Streamlit.

If that's not enough, Wave is more feature-rich:

* More widgets.
* Auth support out of the box without lockin to a custom cloud platform.
* Dev tooling - VScode/Jetbrains plugins to ease development.

The best answer though is to play around a bit and see the differences for yourself.

> One thing I love about panel's design is how easy it is to get something up and running (without using all the pages/cards)

That's fair. Wave has a bit more verbose, but gives you more control over the layout on the other hand (based on very brief look at the docs). You can try `wave init` cli command once you pip install Wave to get up and running in no time.
